### PR TITLE
Update ChineseCalendarHolidays: add Ching Ming Festival

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Update `ChineseCalendarHolidays` (add Ching Ming Festival).

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] This PR is filed against `beta` branch of the repository
- [x] This PR doesn't contain any merge conflicts and has clean commit history
- [x] The code style looks good: `make pre-commit`
- [x] All tests pass locally: `make test`, `make tox` (we strongly encourage adding tests to your code)
- [x] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/holiday_groups.py [hunk 1]
```diff
@@ -662,6 +669,16 @@ def _add_chinese_new_years_day_five(self, name) -> Optional[date]:
             days_delta=+4,
         )
 
+    def _add_ching_ming_festival(self, name) -> date:
```
Comment (arkid15r) [inline holidays/holiday_groups.py:672]: Any objections to use this name instead?
```suggestion
    def _add_qingming_festival(self, name) -> date:
```

**Review by arkid15r — COMMENTED** [review]
LGTM

File: holidays/holiday_groups.py (hunk 1 above)
Comment (PPsyrius) [reply holidays/holiday_groups.py:672]: +1 on this, we should avoid Wade-Giles if possible unless it's better known in the English language that way for standardisation.

Comment (KJhellico) [reply holidays/holiday_groups.py]: Renamed. Holiday name in Hong Kong remained "Ching Ming Festival" as in HK Government source.

**Pull request merged**